app/views/resources/_work_version.html.erb

```diff
@@ -13,25 +13,17 @@
       ) %>
 
   <% if policy(work_version.work).edit? %>
-    <li class="nav-item">
-      <%= render LinkDisabledByTooltipComponent.new(
-            enabled: policy(work_version).edit?,
-            text: I18n.t('resources.work_version.edit_button.text', version: work_version.display_version_short),
-            path: dashboard_form_work_version_details_path(work_version.id),
-            tooltip: I18n.t('resources.work_version.edit_button.tooltip'),
-            class_list: 'btn btn-outline-light btn--squish mr-lg-2 qa-edit-version'
-          ) %>
-    </li>
+    <% if current_user.admin? %>
+      <li class="nav-item">
+        <a class="btn btn-outline-light btn--squish mr-lg-2 qa-edit-version"
+           href="<%= dashboard_form_work_version_details_path(work_version.id) %>">
+          <%= I18n.t('resources.work_version.edit_version_button', version: work_version.display_version_short) %>
```

Comment:
`edit_version` threw me for a little bit. Would `admin_edit_button` make more sense?